src/features/languages/generated/client/descriptions.json

```diff
@@ -241,6 +241,7 @@
     "userSetup.securityQuestions.birthTown": "The description for BIRTH_TOWN key",
     "constants.cha": "The description for CHA type",
     "constants.chairman": "The description for CHAIRMAN type",
+    "settings.message.changeLanguage": "Change language message",
```

Comment:
Yes, do we need french translation?

Reply:
The french translation is actually there. Only the english one was removed so I just added it back